ooici/coi-services
# adjustments related with handling of external OMS events in platform agent code

- generate_test_event implemented in the simulator
- use FQDN for the URL (listener) when testing against real endpoint (so they can resolve the http server)
- initial adjustment related with the identification of platform attributes (based on composition of attr_name and attr_instance)
- use local port 5000 for the event listener (previously 12021): 5000 is more convenient as it is also the default for the gateway_service.
- properly parse listener url for event notification in simulator
- RSNPlatformDriver no longer launches own HTTP server for reception of external OMS events (this functionality is provided by ServiceGatewayService) -- it only registers the corresponding listener URL.
- remove now obsolete dispatch of reception of external event and its publication to the CI in the platform_agent -- this functionality is now provided by the gateway service.
- use DeviceStatusEvent (instead of DeviceEvent) for the internal diagnostics handling

_all_ platform related unit and int tests passing.
